pkgs/by-name/ae/aerospace/package.nix

```diff
@@ -0,0 +1,53 @@
+{
+  fetchzip,
+  gitUpdater,
+  lib,
+  stdenv,
+  versionCheckHook,
+}:
+
+let
+  ghUrl = "https://github.com/nikitabobko/aerospace";
+  # Version must be a literal string as package has a capital letter in it
+  # whilst nixpkgs forbids capitals in version names
+  ghReleaseVersion = "0.14.2-Beta";
+
+  pname = "aerospace";
+  appName = "AeroSpace.app";
+in
+stdenv.mkDerivation {
+  inherit pname;
+
+  version = lib.strings.toLower ghReleaseVersion;
+
+  src = fetchzip {
+    url = "${ghUrl}/releases/download/v${ghReleaseVersion}/AeroSpace-v${ghReleaseVersion}.zip";
+    hash = "sha256-v2D/IV9Va0zbGHEwSGt6jvDqQYqha290Lm6u+nZTS3A=";
+  };
```

Comment:
Use `fetchFromGitHub` instead, and drop the `ghUrl` stuff; `passthru.gitUpdater` will work fine with `fetchFromGitHub`.

Reply:
Ohh thx, I didn't know (first time using gitUpdater)

Reply:
fetchFromGithub breaks the install phase as i'm using the binaries

Reverting to old behaviour